src/main/java/no/entur/antu/validation/NetexValidationProfile.java

```diff
@@ -54,13 +65,28 @@ public ValidationReport validate(
       validationProfile
     );
 
+    ValidationState validationState =
```

Comment:
I would suggest splitting line 68-82 into a separate (or multiple) method(s) since `validate` itself is quite clean as it is today.

Reply:
For example, something like:

```
private ValidationState getValidationState(String validationReportId) {
    return validationStateRepository.getValidationState(validationReportId);
}

private boolean validationAlreadyComplete(String validationReportId) {
  return this.getValidationState(validationReportId) == null;
}

private boolean validationHasErrorInCommonFile(String validationReportId) {
  ValidationState validationState = this.getValidationState(validationReportId);
  if (validationState != null) {
    return validationState.hasErrorInCommonFile();
  }
}
```